seed.js

```diff
@@ -0,0 +1,155 @@
+const mongoose = require('mongoose');
+const Promise = require('bluebird');
+const faker = require('faker');
+
+mongoose.connect('mongodb://localhost/description', {useUnifiedTopology: true, useNewUrlParser: true}).catch(err => console.log(err))
+
+var Schema = mongoose.Schema;
+
+var description = new Schema({
+  item_number: Number,
+  list_date: Date,
+  item_Spec: {
+    condition: String,
+    brand: String,
+    type: String,
+    packaging: String,
+    material: String,
+    rec_age: String,
+    char_family: String,
+    manfactured: String,
+    era: String,
+    year: Number,
+    size: Number,
+    upc: Number
+  },
+  seller_msg: {
+    prod_des: String,
+    item_des: String,
+    img_url: String
+  }
+})
+
+var ship_pay = new Schema ({
+  item_number: Number,
+  ship_handling: {
+    item_location: String,
+    ship_to: String,
+    ship_excludes: String,
+    qty: Number
+  },
+  shipping_cost: {
+    price: Number,
+    region:String,
+    service: String,
+    est_time: String
+  },
+  return_policy: {
+    exist: Boolean,
+    deadline: Number,
+    type: String,
+    pay_shipping: String
+  }
+});
+
+var Product = mongoose.model('Product', description);
+var Purchase = mongoose.model('Purchase', ship_pay)
+
+let seed = () => {
+  let clear = dropdb()
+  let random = randomProduct()
+  let data = [];
+  let transaction = [];
+  for(let i =0; i < 10; i++) {
+    let currProd = {
+      item_number: i,
+      list_date: faker.date.recent(),
+      item_Spec: {
+        condition: faker.lorem.word(),
+        brand: random[0],
+        type: random[1],
+        packaging: random[2],
+        material: random[3],
+        rec_age: random[4],
+        char_family: random[5],
+        manfactured: random[6],
+        era: random[7],
+        year: random[8],
+        size: random[9],
+        upc: random[10]
+      },
```

Comment:
Currently I'm unsure of why the randomProduct function is necessary. It appears as though the logic you applied inside the randomProduct function could be used within this for loop directly to set the values of the different keys inside "item_Spec". Similar to how you applied values in "seller_msg". Was there a reason you separated those items into a new function that I'm not thinking of?

Reply:
I set this up after going over my component plan with Alex. However since it is probably not a common think to do when seeding a data base, I should have included a comment in the code explaining it, will do so in the future.

I created this function to copy what I found in ebay for this part of the component.  From what I found only one of these values that is required when putting a product for sale. This function randomizes what values are present for each item.  If the exists var evaluates to 1 then a lorem is added to the array, if 0 then an empty string is added to the array.  When displaying in the component only items with a lorem value will be displayed and empty strings will not.